sdk/storage_blobs/src/blob/operations/clear_page.rs

```diff
@@ -27,9 +27,9 @@ impl ClearPageBuilder {
             headers.insert(PAGE_WRITE, "clear");
             headers.insert(BLOB_TYPE, "PageBlob");
             headers.add(self.ba512_range);
-            headers.add(self.sequence_number_condition);
-            headers.add(self.if_modified_since_condition);
-            headers.add(self.if_match_condition);
+            headers.add(self.if_sequence_number);
```

Comment:
It would be nice if we could figure out how to apply these headers without having to write this code over and over in different operations.